<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkCaPTkExamplePyControls</class>
 <widget class="QWidget" name="QmitkCaPTkExamplePyControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>857</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>QmitkCaPTkExample1</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QPushButton" name="pushButton_DoStuff">
     <property name="text">
      <string>Do stuff</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_CloneImage">
     <property name="text">
      <string>Clone Image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_InvertImage">
     <property name="text">
      <string>Invert Image (Pure MITK Implementation)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_pythonscript">
     <property name="text">
      <string>Run Python Sample Script (sample_script.py)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_processInPython">
     <property name="text">
      <string>Process in Python (ProcessImage.py)</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
